python bindings install fails if `python` binary is missing

Although `python3` binary was detected as valid interpreter, install still failed due to missing `python` binary:
```
../../py-compile[155]: python: inaccessible or not found
make[4]: *** [Makefile:551: install-pyexecPYTHON] Error 127
```

`Version: 0.8.5`

--- a/bindings/python3/Makefile.am
+++ b/bindings/python3/Makefile.am
@@ -27,7 +27,6 @@ AM_CPPFLAGS = -I. -I$(top_builddir) $(PYTHON3_INCLUDES)
 LIBS = ${top_builddir}/src/libcap-ng.la
 SWIG_FLAGS = -python
 SWIG_INCLUDES = ${AM_CPPFLAGS}
-PYTHON = $(PYTHON3)
 pyexec_PYTHON = capng.py
 pyexec_LTLIBRARIES = _capng.la
 pyexec_SOLIBRARIES = _capng.so

